src/main/AndroidManifest.xml

```diff
@@ -75,5 +74,16 @@
             </intent-filter>
         </receiver>
         -->
+        <activity android:name="com.mixpanel.android.mpmetrics.MixpanelWebViewActivity">
```

Comment:
can we add comments above each of these explaining what they're used for so that developers integrating our library will understand?